doc/modules/cross_validation.rst

```diff
@@ -947,49 +947,51 @@ Permutation test score
-p-value, which represents how likely an observed performance of the
-classifier would be obtained by chance. The null hypothesis in this test is
-that the classifier fails to leverage any statistical dependency between the
-features and the labels to make correct predictions on left out data.
+to evaluate the performance of a model that we can score on. It provides a
+permutation-based p-value, which represents how likely an observed performance of the
+estimator would be obtained by random chance. The null hypothesis in this test is
+that the estimator fails to leverage any statistical dependency between the
+features and the targets to make correct predictions on left out data.
 :func:`~sklearn.model_selection.permutation_test_score` generates a null
 distribution by calculating `n_permutations` different permutations of the
-data. In each permutation the labels are randomly shuffled, thereby removing
-any dependency between the features and the labels. The p-value output
-is the fraction of permutations for which the average cross-validation score
-obtained by the model is better than the cross-validation score obtained by
+data. In each permutation the targets are randomly shuffled, thereby removing
+any dependency between the features and the targets. The p-value output
+is the fraction  of randomized data sets for which the model's average cross-validation
+score on the permutations is better or equal than the cross-validation score obtained by
 the model using the original data. For reliable results ``n_permutations``
 should typically be larger than 100 and ``cv`` between 3-10 folds.
 
-A low p-value provides evidence that the dataset contains real dependency
-between features and labels and the classifier was able to utilize this
-to obtain good results. A high p-value could be due to a lack of dependency
-between features and labels (there is no difference in feature values between
-the classes) or because the classifier was not able to use the dependency in
-the data. In the latter case, using a more appropriate classifier that
-is able to utilize the structure in the data, would result in a lower
-p-value.
-
-Cross-validation provides information about how well a classifier generalizes,
-specifically the range of expected errors of the classifier. However, a
-classifier trained on a high dimensional dataset with no structure may still
+A low p-value provides evidence that the dataset contains some real dependency between
+features and targets **and** that the estimator was able to utilize this dependency to
+obtain good results. A high p-value, in reverse, could be due to either one of these:
+  - a lack of dependency between features and targets (i.e., there is no systematic
+    relationship and any observed patterns are likely due to random chance)
+  - **or** because the estimator was not able to use the dependency in the data (for
+    instance because it under fit).
```

Comment:
I think a blank line at the top may fix it? If still not working, the bullets may need not need to be indented?

Reply:
Doing both in combination worked. :fireworks: